themes/courses.labster.com/lms/templates/ccx/enrollment.html

```diff
@@ -62,12 +63,13 @@ <h2 class="hd hd-2">${_("Batch Enrollment")}</h2>
 </div>
 
 <div class="member-lists-management" style="float:left;width:50%">
-  <form method="POST" action="ccx_manage_student" class="ccx-manage-student-form">
+  <form method="POST" action="ccx_manage_student" class="ccx-manage-student-form" onsubmit="return ccxManageStudentForm(this)">
```

Comment:
@lukyfebrianto can we simply use `<input type="email" />` instead?
See https://developer.mozilla.org/en-US/docs/Web/HTML/Element/input/email